src/test/resources/PatrimoineRicheSupplier.java

```diff
@@ -1,3 +1,5 @@
+package school.hei.patrimoine.cas.drive;
+
```

Comment:
could you explain the use of setting the package name ?

test with a different package name also, to ensure that your code works if user's classes are from different packages

Reply:
done